test/doc-views/TR/Recommendation/REC.js

```diff
@@ -34,6 +34,50 @@ export default {
                 needErrata: false,
             },
         },
+        recWithProposedSubChanges: {
+            ...good,
+            config: {
+                ...good.config,
+                needErrata: false,
+            },
+            sotd: {
+                ...good.sotd,
+                emHTML: `${good.sotd.emHTML} It includes proposed corrections.`,
```

Comment:
Although this works as `getRecMetadata` checks the whole SotD, it would be better to generate a more accurate test. It seems that overriding `sotd.rec.addition` should be better. Same comment applies for the other 3 examples.